docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -98,7 +98,7 @@ endif::[]
 * You have a Kafka instance in {product-kafka} and the instance is in the *Ready* state. To learn how to create a Kafka instance, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}].
 * https://github.com/git-guides/[Git^] is installed.
 * You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[Visual Studio Code^].
-* https://nodejs.org/en/download/[Node.js 14^] is installed. The https://github.com/blizzard/node-rdkafka[node-rdkafka^] client can't run on later versions.
+* https://nodejs.org/en/download/[Node.js 16^] is installed.
```

Comment:
@pacostas Just curious why we removed the noted limitation about node-rdkafka in addition to changing the minimum specified version of Node.js. Is that limitation no longer true?

Reply:
@jbyrne-redhat I changed it from 14 to 16 because on May will go EOL, although I agree keeping it to 14. I think it would be nice to add something like `Node.js 14 or later is installed.` if you think this would help.
Yes, this limitation no longer is true, node-rdkafka can run on node versions 14, 16 and 18.

Reply:
Yes, I agree:

```
* https://nodejs.org/en/download/[Node.js 14^] or later is installed
```